Gemfile

```diff
@@ -53,6 +56,12 @@ gem 'public_activity'
 group :development do
   gem 'better_errors'
   gem 'letter_opener'
+  # Access an interactive console on exception pages or by calling 'console' anywhere in the code.
+  gem 'web-console', '>= 3.3.0'
+  gem 'listen', '>= 3.0.5', '< 3.2'
+  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem 'spring'
```

Comment:
@asideofcode-dev just a quick question, was this needed for the upgrade or did you add it for other reasons (e.g. the performance boost it gives in development mode)

Reply:
I use the `rails app:update` task to reconcile boilerplate files between versions (as illustrated here https://railsdiff.org/4.2.11.3/5.2.8). This gem came from there.

Reply:
Ok, makes sense, thanks for clarifying